fearless/Common/View/Chart/FWBarChartView.swift

```diff
@@ -126,8 +126,15 @@ extension FWBarChartView: FWChartViewProtocol {
                 return chartAmount.filled ? chartAmount.value : 0.0
             }
         }
-        let (min, max) = (realAmounts.min() ?? 0.0, realAmounts.max() ?? 0.0)
-        averageLabelHeightPercent = (data.averageAmountValue - min) / (max - min)
+
+        if
+            let averageAmountValue = data.averageAmountValue,
+            let min = realAmounts.min(),
+            let max = realAmounts.max() {
```

Comment:
If max == min then it will case division by 0 again, maybe it is worth to change it to be:
```swift
If
       let averageAmountValue = data.averageAmountValue,
       let min = realAmounts.min(),
       let max = realAmounts.max(),
       max != min
```

Reply:
Or maybe
```swift
guard let min = realAmounts.min(),
      let max = realAmounts.max(),
      min != max else { ... } // throw an error?
...
```